fivetran_provider/example_dags/example_fivetran.py

```diff
@@ -17,15 +17,15 @@
 
 fivetran_sync_start = FivetranOperator(
     task_id='fivetran-task',
-    fivetran_conn_id='fivetran',
-    connector_id=Variable.get('connector_id'),
+    fivetran_conn_id='fivetran_default',
+    connector_id="{{ var.value.get('connector_id', 'fallback_connector_id') }}",
```

Comment:
If you don't specify the `connector_id`, the sensor should error out and exit. `fallback_connector_id` is probably going to do more harm than good here.